nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -362,7 +362,7 @@ let
 
   # Name used by UEFI for architectures.
   targetArch =
-    if pkgs.stdenv.isi686 then
+    if pkgs.stdenv.isi686 || config.boot.loader.force32Bit then
```

Comment:
I'm a bit confused, why is this option only used in the installer? This then doesn't seem to have any effect on standard (non-installer) Nix systems.